packages/vapor/scss/variables.scss

```diff
@@ -276,26 +276,26 @@ $table-border-width: 1px;
 $table-cell-left-right-padding: 15px;
 $table-header-inner-height: 30px;
 $table-header-height: $table-header-inner-height + 2 * $table-border-width;
-$table-header-font-size: 12px;
+$table-header-font-size: 13px;
 $table-header-line-height: 16px;
 $table-header-top-bottom-padding: (($table-header-inner-height - $table-header-line-height) / 2);
 $table-header-z-index: 2;
 $table-row-inner-height: 36px;
 $table-row-height: $table-row-inner-height + $table-border-width;
-$table-row-font-size: 15px;
+$table-row-font-size: 14px;
 $table-row-line-height: 18px;
 $table-row-top-bottom-padding: (($table-row-inner-height - $table-row-line-height) / 2);
-$table-border: $table-border-width solid var(--table-border-color);
+$table-border: $table-border-width solid var(--grey-40);
 $table-border-button: $table-border-width solid transparent;
 $table-selected-border-width: 5px;
 $table-column-draggable-width: 25px;
 $table-column-draggable-padding: 7px 10px;
-$table-row-drag-over-color: var(--table-row-drag-over-color);
+$table-row-drag-over-color: rgba($medium-grey, $transparency-3);
```

Comment:
old color

Reply:
Yes, I keep the existing value which is on the current UI. https://github.com/coveo/react-vapor/pull/1895/files/d53cc588d7dda25f7a53a3ee1aaa85814f10f927#diff-517a27cf28e30b251089bce6cc3eacd9a2e0a5416c6766e6296f122d565c88e9L272